pkg/plugin/config.go

```diff
@@ -31,6 +31,12 @@ type PluginConfig struct {
 	GitHubAppInstallationID string
 	// The private Key PEM obtained for github app.
 	GitHubAppPrivateKeyPEM string
+
+	// GitHubPlugInDisplayName is for display, e.g. for the web UI.
+	GitHubPlugInDisplayName string
```

Comment:
`GithubPlugin` prefix is added to aviod env var naming collision in case users uses multiple plugins.